src/google/protobuf/stubs/int128_unittest.cc

```diff
@@ -44,16 +44,18 @@ namespace protobuf {
 
 TEST(Int128, AllTests) {
```

Comment:
The test run is showing some errors like this:
```
google/protobuf/stubs/int128_unittest.cc: In member function ‘virtual void google::protobuf::Int128_Traits_Test::TestBody()’:
google/protobuf/stubs/int128_unittest.cc:370:15: error: ‘is_trivially_copy_constructible’ is not a member of ‘std’
   EXPECT_TRUE(std::is_trivially_copy_constructible<uint128>::value);
```
I think you might just need to include `<type_traits>` at the top of this file.

Reply:
Thanks, done.

Reply:
I'm still seeing the same error but I think it's because GCC 4.x apparently doesn't support `std::is_trivially_copy_constructible` or `std::is_trivially_copy_assignable`. Would you mind disabling this test case for GCC versions before 5.0? I think something like this should work:
```
#if !defined(__GNUC__) || __GNUC__ > 4
...
#endif
```

Reply:
done